kms/api/Kms.Samples/IamAddMember.cs

```diff
@@ -35,13 +35,22 @@ public class IamAddMemberSample
         // var resourceName = new KeyRingName(projectId, locationId, keyRingId);
 
         // Get the current IAM policy.
-        Policy policy = client.GetIamPolicy(resourceName);
+        Policy policy = client.IAMPolicyClient.GetIamPolicy(
```

Comment:
Urgh, we should add method signatures accepting resource names...

Reply:
Yep, I thought so as well, are these clients not being generated the same as the others? Is it lack of configuration?